README.md

```diff
@@ -112,6 +113,43 @@ jobs:
       contents: 'read'
       actions: 'read'
 ```
+<!-- end permissions-in-v2 -->
+
+## Self-hosted runners
+
+This Action supports usage of your own self-hosted runners, but since it uses GitHub APIs you will need to grant in explicit access rights:
```

Comment:
```suggestion
This Action supports usage of your own self-hosted runners, but since it uses GitHub APIs you will need to grant it explicit access rights:
```